docusaurus/docs/Android/04-compose/04-message-components/01-messages-screen.mdx

```diff
@@ -53,42 +54,40 @@ fun MessagesScreen(
 Here's an example of setting up custom behavior for the user tapping on the back button and the channel info:
 
 ```kotlin
-    override fun onCreate(savedInstanceState: Bundle?) {
-        super.onCreate(savedInstanceState)
-        val channelId = intent.getStringExtra(KEY_CHANNEL_ID) ?: return
-
-        setContent {
-            ChatTheme {
-                MessagesScreen(
-                    ..., // State
-                    onBackPressed = { finish() }, // Navigation handler
-                    onHeaderActionClick = { channel ->
-                        // Show conversation info
-                    },
-                )
-            }
-        }
-    }
+MessagesScreen(
+    ..., // State
+    onBackPressed = { finish() }, // Navigation handler
+    onHeaderActionClick = { channel ->
+        // Show conversation info
+    },
+)
```

Comment:
Not related to the PR, but maybe we should also invoke the very same action when somebody click on the channel avatar?

Reply:
Or maybe expose another Handler for the channel avatar?